packages/yoastseo/src/scoring/assessments/readability/SubheadingDistributionTooLongAssessment.js

```diff
@@ -127,11 +127,11 @@ class SubheadingsDistributionTooLong extends Assessment {
 
 			const customCountLength = researcher.getHelper( "customCountLength" );
 			const textLength = customCountLength ? customCountLength( paper.getText() ) : researcher.getResearch( "wordCountInText" );
-
-			return paper.hasText() && textLength > this._config.applicableIfTextLongerThan;
+			// Do not use hasEnoughContentForAssessment as it is redundant with textLength > this._config.applicableIfTextLongerThan.
+			return this.hasEnoughContentForAssessment( paper ) && textLength > this._config.applicableIfTextLongerThan;
```

Comment:
Thank you. You are right. I was in doubt, since theoretically it is possible that `hasEnoughCOntentForAssessment` will be changed and not be redundant. But for now I think it is best to leave it out.